### ISSUE TITLE
Create Account

### ISSUE THREAD

**Issue by TylerWasniowski** [TylerWasniowski/tickethub#10]

**Comment by ThePicoNerd** [comment]
Can I take this? Do you mind if I add `bcrypt`, too? Storing the user's password plain in the database is a really, really, really bad habit.

**Comment by ThePicoNerd** [comment]
Fixed with #34, #33 and #32. Watch this video: [How NOT to Store Passwords](https://www.youtube.com/watch?v=8ZtInClXe1Q), I learned a lot when I watched it!

**Comment by TylerWasniowski** [comment]
Yes, it is. Sorry, this is for a class and all of my teammates have almost no experience working on projects. But because this is for a class, we need to do the work. Sorry! Thanks for the offer though!

Just curious by the way, how did you find us?
